statsmodels/genmod/families/family.py

```diff
@@ -529,6 +556,49 @@ def loglike(self, endog, mu, scale=1.):
             return np.sum((endog * mu - mu**2/2)/scale - endog**2/(2 * scale)
                           - .5*np.log(2 * np.pi * scale))
 
+
+    def loglikeobs(self, endog, mu, scale=None):
+        """
+        The log-likelihood in terms of the fitted mean response.
+
+        Parameters
+        ----------
+        endog : array-like
+            Endogenous response variable
+        mu : array-like
+            Fitted mean response variable
+        scale : float, optional
+            Scales the loglikelihood function. The default is 1.
+
+        Returns
+        -------
+        llf : float
+            The value of the loglikelihood function evaluated at
+            (endog,mu,scale) as defined below.
+
+        Notes
+        -----
+        If the link is the identity link function then the
+        loglikelihood function is the same as the classical OLS model.
+        llf = -(nobs/2)*(log(SSR) + (1 + log(2*pi/nobs)))
+        where SSR = sum((endog-link^(-1)(mu))**2)
+
+        If the links is not the identity link then the loglikelihood
+        function is defined as
+        llf = sum((`endog`*`mu`-`mu`**2/2)/`scale` - `endog`**2/(2*`scale`) - \
+            (1/2.)*log(2*pi*`scale`))
+        """
+        #if isinstance(self.link, L.Power) and self.link.power == 1:
+        if scale is None:
+            # This is just the loglikelihood for classical OLS
+            nobs = endog.shape[0]
+            ssr = ss(endog-self.fitted(mu))
```

Comment:
Looks like `ss` is `scipy.stats`.  This is probably intended to be `np.sum(endog - self.fitted(mu)**2, axis=0)`.